modules/web/src/app/node-data/basic/provider/kubevirt/template.html

```diff
@@ -119,7 +119,7 @@
                (changed)="onSubnetChange($event)"
```

Comment:
We also need this `valueFormatter` if we want to display custom value for selected subnet. This would require a change in `getSubnetOptionName` method so it will be passed a `string` value of selected subnet name and then we need to find the subnet from `subnets` array based on this name. Here's a similar example: https://github.com/kubermatic/dashboard/blob/main/modules/web/src/app/node-data/basic/provider/aws/component.ts#L221

```suggestion
               (changed)="onSubnetChange($event)"
               [valueFormatter]="getSubnetOptionName.bind(this)"
```